repondeur/tests/test_lectures_add.py

```diff
@@ -279,6 +288,10 @@ def test_post_form_already_exists(
     assert resp.status_code == 200
     assert "Cette lecture existe déjà…" in resp.text
 
+    with transaction.manager:
```

Comment:
We probably don’t need to wrap this in a DB transaction, as there will be no changes to commit

Reply:
Without the transaction I have a `sqlalchemy.orm.exc.DetachedInstanceError` when we access events.

Reply:
Even with just the DBSession.add?

Reply:
Ah! You're right